src/redux/boards/actions.js

```diff
@@ -94,6 +97,34 @@ const actionCreators = {
       type: actions.DELETE_BOARD_FAILURE,
       payload: { error }
     };
+  },
+  addBoard(boardName, boardColor) {
+    return async dispatch => {
+      dispatch({ type: actions.ADD_BOARD });
+      try {
+        const response = await BoardService.addBoard(boardName, boardColor);
+        if (response.ok) {
+          dispatch(actionCreators.addBoardSuccess());
+          dispatch(actionCreators.getBoards());
+        } else {
+          throw new Error(response);
```

Comment:
Do we need to `throw` here ? Why not just
```
if (response.ok) {
  doStuff();
} else {
  dispatch(actionCreators.addBoardFailure(response));
}
```
?

Reply:
this way you avoid duplicating the dispatch here and in the `catch` block